reader/views.py

```diff
@@ -3250,6 +3250,20 @@ def topic_pool_api(request, pool_name):
     return jsonResponse(response, callback=request.GET.get("callback", None))
 
 
+@catch_error_as_json
+def featured_topic_api(request):
```

Comment:
why enfpoing is featured_topic and not topic_of the day?

Reply:
Mickey has renamed this feature "Featured Topic"